src/fides/api/models/privacy_request.py

```diff
@@ -637,11 +637,27 @@ def cache_manual_webhook_input(
         parsed_data = manual_webhook.fields_schema.parse_obj(input_data)
 
         cache.set_encoded_object(
-            f"WEBHOOK_MANUAL_INPUT__{self.id}__{manual_webhook.id}",
+            f"WEBHOOK_MANUAL_ACCESS_INPUT__{self.id}__{manual_webhook.id}",
             parsed_data.dict(),
         )
 
-    def get_manual_webhook_input_strict(
+    def cache_manual_webhook_erasure_input(
```

Comment:
i know we're following the example from the access input as a first pass, and i think that's probably fine. but i'd want to make sure we flag this, since i could see it leading to problems if users (or our product team) doesn't realize the implication here.

specifically, there's no sort of persisted record of the confirmation that the records have been deleted from the manual integration. there'd be no way of tracking that admin X confirmed at time Y that person Z's records were deleted from system A (whoops ran out of letters) -- at least, not one that we can rely on since we have to treat the cache as ephemeral. i think it's true that we don't persist directly in our db the manual input data for access requests (though maybe we should?) but it's a bit different there because that input is packaged up as part of the access result package and that package is put into storage and/or sent back to the end user -- so it _is_ persisted somewhere.

this is certainly not listed out as an explicit requirement here, but it feels like a reporting/audit requirement that could potentially have legal implications if/when this gets implemented. not saying we need to solve it here in a first pass, i'd just want this to be flagged before we release!

Reply:
We do not need a persisted record of the confirmation since a successfully completed deletion requires the confirmation we can infer that a confirmation was made. CC @rsilvery

Reply:
OK, that's fine for me in terms of an MVP, but there's some tension with the other piece of this implementation as noted [here](https://github.com/ethyca/fides/pull/4115#discussion_r1337661024). 

maybe we can ticket for a follow up, just to track it at least?

Reply:
Placeholder ticket https://github.com/ethyca/fides/issues/4164